BridgeSDK/BridgeAPI/SBBParticipantManager.m

```diff
@@ -40,6 +40,21 @@ + (instancetype)defaultComponent
     return shared;
 }
 
++ (NSArray<NSString *> *)dataSharingScopeStrings
+{
+    static NSArray<NSString *> *scopeStrings = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        scopeStrings =  @[
+                          kSBBParticipantDataSharingScopeStrings[0],
+                          kSBBParticipantDataSharingScopeStrings[1],
+                          kSBBParticipantDataSharingScopeStrings[2]
+                          ];
```

Comment:
This feels like a maintenance crash waiting to happen. Could you please build using `sizeof`?